tests/test_ecommerce.py

```diff
@@ -27,7 +33,18 @@
 
 from . import get_crawler
 from .test_utils import URL_TO_DOMAIN
-from .utils import assertEqualSpiderMetadata, get_addons
+from .utils import assertEqualSpiderMetadata, crawl_fake_zyte_api, get_addons
+
+if TYPE_CHECKING:
+    from aiohttp.pytest_plugin import AiohttpServer
+
+
+@pytest_twisted.async_fixture(scope="module")
```

Comment:
why not to use "session" scope?

Reply:
The options are "function" and "module".